.github/workflows/actions.yml

```diff
@@ -89,3 +71,18 @@ jobs:
         with:
           command: fmt
           args: --all -- --check
+
+  docs:
```

Comment:
@cloudhead Do we even need `cargo doc` in CI?  I don't think it fails the CI job even if there are egregious syntax errors:

```
$ env RUSTFLAGS="-D warnings" cargo doc --profile ci --no-deps
[...]
warning: unresolved link to `fdsf`
 --> push/src/lib.rs:9:6
  |
9 | /// [`fdsf]
  |      ^^^^^ no item named `fdsf` in scope
  |
  = note: `#[warn(rustdoc::broken_intra_doc_links)]` on by default
  = help: to escape `[` and `]` characters, add '\' before them like `\[` or `\]`

warning: `rad-push` (lib doc) generated 1 warning
    Finished ci [unoptimized] target(s) in 2m 22s
% echo $?
0
```

Reply:
Correction: I misread the code: It says `RUSTDOCFLAGS`, not `RUSTFLAGS` and it does return a non-zero exit code as is desired in CI.